hammr/utils/hammr_utils.py

```diff
@@ -270,3 +270,29 @@ def extract_scan_id(image_uri):
         return int(match.group(1))
     else:
         return None
+
+def extract_appliance_id(image_uri):
+    match = re.match( r'users/[^/]+/appliances/([0-9]+)($|/)', image_uri)
+    if match:
+        return int(match.group(1))
+    else:
+        return None
+
+def check_mandatory_migration(migration):
+    return migration
+
+def check_mandatory_migrate(migration):
```

Comment:
could be move to migration_utils isn't it?

Reply:
I moved to migration_utils